pkg/bpfassets/attacher/bcc_attacher.go

```diff
@@ -138,9 +139,24 @@ func AttachBPFAssets() (*BpfModuleTables, error) {
 	}
 	// TODO: verify if ebpf can run in the VM without hardware counter support, if not, we can disable the HC part and only collect the cpu time
 	m, err := loadModule(objProg, options)
+	if err != nil {
+		klog.Infof("failed to attach perf module with options %v: %v, from default kernel source.\n", options, err)
+		dirs := config.GetKernelSourceDir()
```

Comment:
nit: better rename it to SourceDirs if we allow multiple dirs ..

Reply:
+1 to the suggestion above and if you follow https://go.dev/doc/effective_go#Getters  idiom, the recommended name would be `config.KernelSourceDirs()` 👼